exodus/men_customization/templates/base/home.html

```diff
@@ -0,0 +1,178 @@
+{% extends "base/base.html"%}
+{% block content %}
+{% load i18n %}
+{% load static %}
+<div class="row justify-content-center mb-5 mt-5">
+  <div class="col-xl-3 col-lg-3 col-md-3 col-sm-4 col-4 my-auto">
+    <img src="{% static "men_customization/logo-amal.png" %}" class="mx-auto d-block img-fluid" alt="amal logo">
```

Comment:
Is it OK to have the MEN logo in our repository?

Reply:
Yes it seems to be okay.